### DISCUSSION TITLE
请问 win10 2019 ltsc 如何安装使用HyPlayer

### DISCUSSION THREAD

**Discussion by ghost** [HyPlayer/HyPlayer#62] [Q&A] [answered]
安装提示系统版本不符合要求
![image](https://user-images.githubusercontent.com/78802618/126900531-3e28d65c-24b1-474b-9d40-0db637b1637e.png)
![image](https://user-images.githubusercontent.com/78802618/126900435-25671e2f-a2d3-4dc9-989d-6ecb1c6fef61.png)

请教一下开发者，有没有办法在这个系统版本上使用Hyplayer呢？

**Comment by EP012014** [comment, marked as answer]
17763不行，最低支持18362（1903）

**Reply by ghost** [reply]
好的，谢谢。看来得等今年的2021 ltsc 版本了。
